sdk/exportentities/workflow_notif.go

```diff
@@ -0,0 +1,208 @@
+package exportentities
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/ovh/cds/sdk"
+)
+
+// Booleans
+var (
+	True  = true
+	False = false
+)
+
+// NotificationEntry represents a notification set on a nodeEntry
+type NotificationEntry struct {
+	Type     string                        `json:"type" yaml:"type"`
+	Settings *sdk.UserNotificationSettings `json:"settings,omitempty" yaml:"settings,omitempty"`
+}
+
+// craftNotificationEntry returns the NotificationEntry and the name of the nodeEntries concerned
+func craftNotificationEntry(w sdk.Workflow, notif sdk.WorkflowNotification) ([]string, NotificationEntry, error) {
+	entry := NotificationEntry{}
+	nodeNames := make([]string, len(notif.SourceNodeIDs))
+	for i, id := range notif.SourceNodeIDs {
+		node := w.GetNode(id)
+		if node == nil {
+			return nil, entry, fmt.Errorf("workflow node %d not found", id)
+		}
+		nodeNames[i] = node.Name
+	}
+	sort.Strings(nodeNames)
+	entry.Type = notif.Type
+	entry.Settings = &notif.Settings
+
+	// Replace the default values by nil
+	if entry.Settings.OnStart != nil && !*entry.Settings.OnStart {
+		entry.Settings.OnStart = nil
+	}
+	if entry.Settings.SendToGroups != nil && !*entry.Settings.SendToGroups {
+		entry.Settings.SendToGroups = nil
+	}
+	if entry.Settings.SendToAuthor != nil && *entry.Settings.SendToAuthor {
```

Comment:
```suggestion
	if entry.Settings.SendToAuthor != nil && !*entry.Settings.SendToAuthor {
```

Reply:
nop because the default behavoir is to send to the author.
so if entry.Settings.SendToAuthor is not nil and is true, it doesn't have to be displayed